bcipy/demo/bci_main_demo.py

```diff
@@ -22,10 +22,9 @@ def main():
 
     # Try and initialize with bci main
     try:
```

Comment:
Seems like it might be fine to drop the `try/except` block here, since catching the exception only results in printing it out